backend/apps/owasp/management/commands/owasp_run_notification_worker.py

```
@@ -0,0 +1,113 @@
+        """Handle snapshot published event."""
+        try:
+            snapshot_id = int(data.get(b"snapshot_id").decode("utf-8"))
+            snapshot = Snapshot.objects.get(id=snapshot_id)
+            
+            users = User.objects.filter(is_active=True)
+            
+            if not users.exists():
+                logger.info("No active users found.")
+                return
+
+            logger.info(f"Sending snapshot notification to {users.count()} users")
```

Ruff (0.14.14)

[warning] Logging statement uses f-string

(G004)